### PR TITLE
:rocket: Send logs to Splunk in JSON

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-193.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-193.dev.beta.nhschoices.net)

File: rancher-config/docker-compose.yml [hunk 1]
```diff
@@ -17,3 +17,14 @@ services:
...
       io.rancher.container.pull_image: always
       io.rancher.scheduler.affinity:host_label_soft: c2s=true
+    logging:
+      driver: splunk
+      options:
+        splunk-url: ${SPLUNK_HEC_URL}
+        splunk-token: ${SPLUNK_HEC_TOKEN}
+        splunk-insecureskipverify: "true"
+        splunk-sourcetype: docker
+        splunk-source: profiles
```
Comment (adamgraves-choices) [inline rancher-config/docker-compose.yml]: This should to be `gp-finder`

Comment (st3v3nhunt) [inline rancher-config/docker-compose.yml]: Updated

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-193.dev.beta.nhschoices.net)

**Pull request merged**